### ISSUE TITLE
Error at the time elapsed.

### ISSUE THREAD

**Issue by sasagar** [coskuncayemre/vue3-flip-countdown#3]
When the time elapsed, the error below would be shown.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toString')
    at Proxy.twoDigits (vue3-flip-countdown.common.js?243e:25699)
    at eval (vue3-flip-countdown.common.js?243e:25472)
    at renderList (runtime-core.esm-bundler.js?5c40:5798)
    at Proxy.render (vue3-flip-countdown.common.js?243e:25459)
    at renderComponentRoot (runtime-core.esm-bundler.js?5c40:464)
    at ReactiveEffect.componentUpdateFn [as fn] (runtime-core.esm-bundler.js?5c40:4400)
    at ReactiveEffect.run (reactivity.esm-bundler.js?a1e9:160)
    at callWithErrorHandling (runtime-core.esm-bundler.js?5c40:6737)
    at flushJobs (runtime-core.esm-bundler.js?5c40:6976)
```

**Comment by sasagar** [comment]
Looking at the code, it looks like the parts that were fixed in the 1.0.4 commit are not reflected in the download file from npm.
It also looks like `emit` is not working.

**Comment by coskuncay** [comment]
@sasagar sorry about confusion, you can try again. :)
